### PR TITLE
Create offboarding.md

### PR DESCRIPTION
**Pull request by its-a-lisa-at-work**
first pass at the offboarding checklist; might not be perfect

### CODE REVIEW

**Comment by its-a-lisa-at-work** [comment]
hmmm, I don't know what I need to do with this..

File: .github/ISSUE_TEMPLATE/offboarding.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Initiate the process via the Leaving TTS Handbook page
+
+Assignee
+- [ ] Remove from IaaS
+  - [ ] AWS
+  - [ ] GCP
+  - [ ] Azure 
+- [ ] [Remove them to the @tech-portfolio user group in Slack.](https://get.slack.help/hc/en-us/articles/212906697-User-Groups#edit-a-user-group)
+- [ ] Remove them to the recurring TTS Tech Portfolio meetings that are relevant for them in [the team calendar](https://calendar.google.com/calendar?cid=Z3NhLmdvdl82aDI5YnF1ZGx0NGVoZTVzOWswOGlmdGZxMEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
+- [ ] Remove them to the [@18F/tts-tech-portfolio](https://github.com/orgs/18F/teams/tts-tech-portfolio) team on GitHub.
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/offboarding.md:26]: These would all be taken care of as part of the [general offboarding tasks](https://docs.google.com/spreadsheets/d/1rp8n78tFHqyvvHzsYuBo_XE3Jg1LwOuOX-R9TMRnZI8/edit#gid=1916685737).

```suggestion
```

File: .github/ISSUE_TEMPLATE/offboarding.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Assignee: The tasks below are organized by the role needed to complete them. If you canʼt complete
+any of the items on your checklist personally, you are responsible for ensuring that an appropriate person
+does it.
+For compliance tracking purposes we need to show when offboarding actions happened; many need to
+occur in a 24-hour window. As you do this work, comment on this GitHub issue as you complete each
+task. (PS-4: Personnel termination)
+
+Departing Person
+- [ ] Initiate the process via the Leaving TTS Handbook page
```
Comment (afeld) [inline .github/ISSUE_TEMPLATE/offboarding.md:17]:
```suggestion
- [ ] [Initiate the process](https://handbook.tts.gsa.gov/leaving-tts/)
```

**Review by afeld — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/offboarding.md (hunk 1 above)
Comment (its-a-lisa-at-work) [reply .github/ISSUE_TEMPLATE/offboarding.md:26]: I don't think so... I feel like those are unique to our team, right?

Comment (afeld) [reply .github/ISSUE_TEMPLATE/offboarding.md:26]: The general offboarding would make these team-specific offboarding steps irrelevant. For example, [Slack user groups can only contain full members](https://slack.com/help/articles/212906697-Create-a-user-group#create-a-user-group), so someone being converted to a single-channel guest would take care of that step. Ditto for the GitHub team. IaaS account removal is in the [offboarding checklist](https://docs.google.com/spreadsheets/d/1rp8n78tFHqyvvHzsYuBo_XE3Jg1LwOuOX-R9TMRnZI8/edit#gid=1916685737) (just added Azure and Google Cloud), so all set there too.

Comment (its-a-lisa-at-work) [reply .github/ISSUE_TEMPLATE/offboarding.md:26]: Ok, got it! I hadn't seen Azure and Google get added to the off-boarding list.

**Review by its-a-lisa-at-work — COMMENTED** [review]
LGTM (finally :))

**Comment by its-a-lisa-at-work** [comment]
@afeld if you could figure out this CircleCI error, that would be helpful, I won't be able to get to this until Monday at the earliest (when I have a monitor)

**Review by afeld — APPROVED** [review]
CircleCI error is `No configuration was found in your project`, which just means that this branch is old. Nothing to worry about.

**Pull request merged**